lib/site_encrypt/certbot.ex

```diff
@@ -12,6 +15,7 @@ defmodule SiteEncrypt.Certbot do
     end
   end
 
+  @spec ensure_cert(map) :: tuple()
```

Comment:
Also, overly generic types. The input is `SiteEncrypt.config`, and the result is `{:new_cert, String.t} | {:no_change, String.t} | {:error, String.t}`.